libs/coin-framework/src/account.test.ts

```diff
@@ -18,6 +18,7 @@ setSupportedCurrencies(["ethereum", "ethereum_classic", "tron"]);
 describe("groupAccountOperationsByDay", () => {
   test("basic", () => {
     const account = genAccount("seed_7", {
+      currency: getCryptoCurrencyById("zcash"),
```

Comment:
the genAccount generates an account with a currency randomly.
when we add new currencies in coin-framework/src/mocks/account.ts, it generates an account with a different currency. this is why I add ```currency: getCryptoCurrencyById("zcash"),```
to avoid the randomness